master/buildbot/reporters/github.py

```diff
@@ -136,12 +137,16 @@ def send(self, build):
         else:
             issue = None
 
-        if project:
+        if "/" in project:
```

Comment:
This was for when I set project to "foobar"  not "owner/repo" This protects the code and allow the code to use the meta data that was really sent to get the data. There is no documentation other than that the project is a meta value. there is nothing saying that it can have magic values. I been burned by this a few times.